src/main/features/certificateOfService/routes/what-documents.ts

```diff
@@ -30,6 +30,14 @@ export default express.Router()
         user.legalUploadDocumentDraft.document.fileToUpload = undefined
         await new DraftService().save(res.locals.user.legalUploadDocumentDraft, res.locals.user.bearerToken)
 
+        if (res.locals.user.legalUploadDocumentDraft && res.locals.user.legalUploadDocumentDraft['id']) {
+          await new DraftService()['delete'](res.locals.user.legalUploadDocumentDraft['id'], res.locals.user.bearerToken)
```

Comment:
is it possible to call like `new DraftService().delete(...)` ?

Reply:
they shouldn't have been there. was in there for testing. Have removed